data/test_bmi_multi_realization_config_w_netcdf.json

```diff
@@ -0,0 +1,154 @@
+                                "registration_function": "register_bmi_cfe",
+                                "variables_names_map": {
+                                    "water_potential_evaporation_flux": "ETRAN",
+                                    "atmosphere_air_water~vapor__relative_saturation": "SPFH_2maboveground",
+                                    "land_surface_air__temperature": "TMP_2maboveground",
+                                    "land_surface_wind__x_component_of_velocity": "UGRD_10maboveground",
+                                    "land_surface_wind__y_component_of_velocity": "VGRD_10maboveground",
+                                    "land_surface_radiation~incoming~longwave__energy_flux": "DLWRF_surface",
+                                    "land_surface_radiation~incoming~shortwave__energy_flux": "DSWRF_surface",
+                                    "land_surface_air__pressure": "PRES_surface",
+				    "ice_fraction_schaake" : "sloth_ice_fraction_schaake",
+				    "ice_fraction_xinan" : "sloth_ice_fraction_xinan",
+				    "soil_moisture_profile" : "sloth_smp"
+                                },
+                                "uses_forcing_file": false
+                            }
+                        }
+                    ],
+                    "uses_forcing_file": false
+                }
+            }
+        ],
+        "forcing": {
+            "path": "./data/forcing/cats-27_52_67-2015_12_01-2015_12_30.nc",
+            "provider": "NetCDF"
+        }
+    },
+    "time": {
+        "start_time": "2015-12-01 00:00:00",
+        "end_time": "2015-12-30 23:00:00",
+        "output_interval": 3600
+    },
+    "catchments": {
+        "cat-27": {
+          "formulations": [
+                        {
+                            "name": "bmi_c",
+                            "params": {
+                                "model_type_name": "bmi_c_pet",
+                                "library_file": "./extern/evapotranspiration/evapotranspiration/cmake_build/libpetbmi",
+                                "forcing_file": "",
+                                "init_config": "./data/bmi/c/pet/cat-27_bmi_config.ini",
+                                "allow_exceed_end_time": true,
+                                "main_output_variable": "water_potential_evaporation_flux",
+                                "registration_function": "register_bmi_pet",
+                                "variables_names_map": {
+                                    "water_potential_evaporation_flux": "potential_evapotranspiration"
+                                },
+                                "uses_forcing_file": false
+                            }
+                        }
+            ],
+            "forcing": {
+                "path": "./data/forcing/cats-27_52_67-2015_12_01-2015_12_30.nc",
+                "provider": "NetCDF"
+            }
+        }
+    },
+    "catchments": {
```

Comment:
This makes two `"catchments"` keys in the same JSON dictionary, which is invalid... at the least, it seems that the formulation for `cat-52` was never executed--because when you fix this, that formulation fails because `sloth_ice_fraction_schaake` is not found in this formulation.

To fix the invalid JSON, remove lines 117 and 118, and add a comma to the end of line 116. After that the formulation below needs to be fixed... this is somewhat tangential to the actual PR, but this is an invalid test.